src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/RcaController.java

```diff
@@ -255,11 +268,42 @@ private void readRcaEnabledFromConf() {
         });
   }
 
+  /**
+   * Reads the mutedRCAs value from the conf file.
```

Comment:
we should also add to the doc comments, what is a valid format for this file ?

Reply:
Right, adding this information.